recipes/georaster/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "georaster" %}
+{% set version = "1.15" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: 72f2e8c57aee8cb01b689025f75a0d90
+
+build:
+    number: 0
+    script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - numpy
+    - scipy
+    - gdal
+    - pyproj
+
+  run:
+    - python
+    - numpy
+    - scipy
+    - gdal
+    - pyproj
+
+test:
+  imports:
+    - georaster
+
+about:
+  home: https://github.com/atedstone/georaster
+  license: GNU GPL v.3
+  summary: 'easy processing and analysis of geographic and projected rasters'
+  license_family: GPL3
+  license_file: LICENSE.txt
```

Comment:
Change this to: `license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE.txt'`. It's currently looking in the source directory from the download.